src/Microsoft.AspNetCore.ResponseCaching/ResponseCachingMiddleware.cs

```diff
@@ -219,10 +214,11 @@ public class ResponseCachingMiddleware
             return false;
         }
 
-        internal async Task FinalizeCacheHeadersAsync(ResponseCachingContext context)
+        private bool FinalizeCacheHeadersShouldStoreVaryByEntry(ResponseCachingContext context)
```

Comment:
This name is a bit silly, a doc comment would be sufficient to explain the return parameter.